ws/sha1.ts

```diff
@@ -47,17 +50,20 @@ export class Sha1 {
 
     this.block = this.start = this.bytes = this.hBytes = 0;
     this.finalized = this.hashed = false;
-    this.first = true;
   }
 
-  update(data: string | ArrayBuffer) {
+  update(data: string | ArrayBuffer | ArrayBufferView) {
     if (this.finalized) {
-      return;
+      return this;
```

Comment:
Not familiar with standards/convention, but why change the return type to `this`?  While it isn't in the style guide yet, Ry and I had a chat about avoiding chaining in our APIs as a default.  If this is aligned to an existing standard, would like to understand it better, otherwise would prefer to leave the return type as `void`.

Reply:
I added `this` for consistency instead of this return type will be inferred as `Sha1 | void`. See this line: https://github.com/denoland/deno_std/blob/6bd349e633a0369359eb503cdbb2815aef30bb14/ws/sha1.ts#L129